core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v3/RdsConfigV3Endpoint.java

```diff
@@ -55,4 +57,18 @@
             nickname = "deleteRdsConfigInOrganization")
     RDSConfigResponse deleteInOrganization(@PathParam("organizationId") Long organizationId, @PathParam("name") String name);
 
+    @POST
+    @Path("testconnect")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = OperationDescriptions.RdsConfigOpDescription.POST_CONNECTION_TEST, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
+            nickname = "testRdsConnection")
+    RdsTestResult testRdsConnection(@PathParam("organizationId") Long organizationId, @Valid RDSTestRequest rdsTestRequest);
+
+    @GET
+    @Path("{name}/request")
```

Comment:
It turns out the order of methods matter for swagger during generating the swagger document. Can you please check if all the endpoints can be seen in the swagger json? 
```https://<your cloudbreak host>/cb/api/swagger.json```

If not, then please reorder methods like:
/v3/{organizationId}/rdsconfigs
...
/v3/{organizationId}/rdsconfigs/{name}
...
/v3/{organizationId}/rdsconfigs/testconnect
...
etc

Reply:
I have reordered methods, but swagger generated them using another ordering. All of them can be found in the swagger json.